File: app/src/main/java/com/example/getfood/ui/loginregister/loginfragment/LoginMvpView.java
package com.example.getfood.ui.loginregister.loginfragment;

import android.content.Context;

import com.example.getfood.ui.base.BaseView;

public interface LoginMvpView extends BaseView {

    Context getContext();

    void valueEntryError(String errMsg);

    void userVerifiedSuccessfully();

    void onUserEmailVerificationFailed();

    void onPasswordResetEmailSentSuccessfully();
}

File: app/src/main/java/com/example/getfood/ui/loginregister/loginfragment/LoginMvpView.kt
package com.example.getfood.ui.loginregister.loginfragment

import android.content.Context
import com.example.getfood.ui.base.BaseView

interface LoginMvpView : BaseView {
    val fragmentContext: Context
    fun valueEntryError(errMsg: String?)
    fun userVerifiedSuccessfully()
    fun onUserEmailVerificationFailed()
    fun onPasswordResetEmailSentSuccessfully()
    fun onTokenUpdatedSuccessfully()
}